lib/Bio/EnsEMBL/DataCheck/Checks/CheckMSANames.pm

```diff
@@ -0,0 +1,79 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckMSANames;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'CheckMSANames',
+  DESCRIPTION    => 'Ensure that every MSA method has a tag name in species_set_header',
+  GROUPS         => ['compara', 'compara_multiple_alignments'],
+  DATACHECK_TYPE => 'critical',
+  DB_TYPES       => ['compara'],
+  TABLES         => ['method_link', 'method_link_species_set', 'species_set_header']
+};
+
+sub skip_tests {
+    my ($self) = @_;
+    my $mlss_adap = $self->dba->get_MethodLinkSpeciesSetAdaptor;
+    my @methods = qw( EPO EPO_LOW_COVERAGE );
```

Comment:
In the HC the filter is based on the `method_link.class`, so that it captures all new types we may want to have in the future (incl. https://www.ebi.ac.uk/panda/jira/browse/ENSINT-334)

Reply:
Also, quite a number of DCs start having a `skip_tests` block that looks for the existence of MLSSs. It will probably help moving that functionality to a new class all those tests could inherit from. But maybe let's wait all the DCs are in to spec that out